libraries/rush-lib/package.json

```diff
@@ -57,7 +57,8 @@
     "tapable": "2.2.1",
     "tar": "~6.1.11",
     "true-case-path": "~2.2.1",
-    "uuid": "~8.3.2"
+    "uuid": "~8.3.2",
+    "pnpm-sync-lib": "~0.0.1"
```

Comment:
Because this package is still experimental, use a locked version `"0.0.1"` to avoid accidental regressions.  (Actually the SemVer standard defines `"~0.0.x"` to be [already locked](https://lfx.rushstack.io/pages/concepts/semver/#semver-surprises), but better to be clear and explicit.)